### PR TITLE
[SMALLFIX]Update Command-Line-Interface.md

### PR DESCRIPTION
**Pull request by njustl**

### CODE REVIEW

File: docs/cn/basic/Command-Line-Interface.md [hunk 1]
```diff
@@ -404,28 +510,34 @@ echo $?
 
 使用举例：使用`touch`命令可以创建一个空文件用于标记一个文件夹的分析任务完成了。
 
-{% include Command-Line-Interface/touch.md %}
-
+```bash
+./bin/alluxio fs touch /data/yesterday/_DONE_
+```
 ### unmount
-
```
Comment (aaudiber) [inline docs/cn/basic/Command-Line-Interface.md]: can we put back this blank line?

File: docs/cn/basic/Command-Line-Interface.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿---
+---
```
Comment (aaudiber) [inline docs/cn/basic/Command-Line-Interface.md]: please undo this whitespace change

Comment (apc999) [reply docs/cn/basic/Command-Line-Interface.md]: not sure what the whitespace it reverts. But I copied the original text and it still shows. I think the new version should be good

**Comment by apc999** [comment]
alluxio-bot, merge this please.

**Pull request merged**